client/src/DiscPanel/DiscPanel.js

```diff
@@ -39,27 +39,34 @@ class DiscPanel extends Component {
     }
 
     render(){
+        let discInfo = '';
+        if (this.state.discInfo.tracks && this.state.discInfo.tracks.length > 0) {
```

Comment:
I needed to conditionally display `<DiscInfo />`. If DiscInfo's constructor fired without `tracks` available, I got some wonky behavior.

Reply:
Better way to do this would be something like:

```
discInfo = () => {
  if(...) {
    return null
  }
  return <DiscInfo ... />
}
```

Then just `discInfo()` when using